lms/services/lti_grading/_v13.py

```diff
@@ -63,6 +63,9 @@ def record_result(self, grading_id, score=None, pre_record_hook=None):
             "activityProgress": "Completed",
             "gradingProgress": "FullyGraded",
         }
+        if comment:
```

Comment:
We only send the comment value is it's present, we don't want to override the LMS value with "nothing".